app/models/todo_item.rb

```diff
@@ -1,6 +1,17 @@
 class TodoItem < ApplicationRecord
   belongs_to :todo_list
+
+
+
+
+
+
+
+
+  
```

Comment:
What for?

Reply:
It was unintentional, will remove that. Thank you.